judge-d-agent/src/test/functional/com/hltech/judged/agent/K8sServiceLocatorFT.groovy

```diff
@@ -57,6 +54,7 @@ class K8sServiceLocatorFT extends Specification {
         wireMockRule.stop()
     }
 
+    @DirtiesContext(methodMode = DirtiesContext.MethodMode.BEFORE_METHOD)
```

Comment:
Why do we need it?

Reply:
After migration system properties required for this test were setting after spring context loaded, it was one possible workaraund, replaced with setting those properties in static initialization block